### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by luisvsilva** [pushlink/pushlink-phonegap#10]
Followed Instructions:

```
var apiKey = "xxxxxxxxxxxx";
pushlink.addMetadata("cordova_version", device.cordova);
pushlink.setCurrentStrategy(pushlink.FRIENDLY_POPUP, {
       popUpMessage: "An Upgrade is available!",
       updateButton: "Upgrade Now"
            });
pushlink.start("mobileapp.nta", apiKey, device.uuid,
            ()=>{  
                console.log("PushLink Started with device ID:" + device.uuid + " and cordo");
            },(error)=> {
                console.error("PushLink start error!" + error);
});
```
And it seems to work, i can see the success message on the console (PushLink Started with device ID:asaslkjsjd87).

But, no device is registered on the backend.

On Logcat:

```
07-07 16:54:51.511 10290 10344 I PushLinkPlugin: initializing plugin v5.5.0
07-07 16:54:51.519 10290 10290 E PUSHLINK: PushLink not initialized. None of the parameter can be null, zero or empty

```
Cordova version is 7.0.1
And android version 6.0.1

**Comment by luisvsilva** [comment]
OK So issue was,  application icon was not being found here:

`final int appIconId = cordova.getActivity().getResources().getIdentifier(ICON, "drawable", packageName);
`
Cordova now inserts icons as mipmap, so trying to get a drawable resource for icon results in zero, which crashes the pushlink plugin.

I have prepared a pull request with this fix, so now it checks for icon in drawable and mipmap, also I have removed the package name parameter because that can be extracted from the Activity object and therefore avoiding potential human error, hope this is ok, if not then let me know and I can pop it back in.

[https://github.com/pushlink/pushlink-phonegap/pull/11](https://github.com/pushlink/pushlink-phonegap/pull/11)

Let me know if you consider merging this, otherwise I will have to run with my fork.

Thanks

Luis
